configs/openmrs_config/concepts/past_medical_history.csv

```diff
@@ -7,6 +7,8 @@ c311aec0-dd3d-4c84-a4ab-601fab82ca37,,,Antihypertensive medication use,បច្
 157621ff-16ac-4dd4-bc96-6164d3248807,,,Premature heart disease/stroke in first degree relatives,ធ្លាប់មានប្រវត្តិជំងឺបេះដូង ឬគ្រោះថ្នាក់សរសៃឈាមខួរក្បាលចំពោះសាច់ញាតិបង្កើត  (ខ្សែទីមួយ),Premature heart disease/stroke in first degree relatives,ធ្លាប់មានប្រវត្តិជំងឺបេះដូង ឬគ្រោះថ្នាក់សរសៃឈាមខួរក្បាលចំពោះសាច់ញាតិបង្កើត  (ខ្សែទីមួយ),,,Question,Coded,Yes;No,,,
 ad8f9ff4-63f1-40b9-b405-aef4ae674c2d,,,Past Medical History,ប្រវត្តិសុខភាពពីអតីតកាល,Past Medical History,ប្រវត្តិសុខភាពពីអតីតកាល,,,ConvSet,N/A,,Known hypertensive;Known diabetic;Antihypertensive medication use;Antidiabetic medication use;Diabetes/Kidney disease in first degree relatives;Premature heart disease/stroke in first degree relatives,,
 e0fffb24-5973-40a9-af11-2e4033831747,,,Presence of TB symptoms,មានរោគសញ្ញាសង្ស័យរបេង,Presence of TB symptoms,មានរោគសញ្ញាសង្ស័យរបេង,,,Question,Coded,Yes;No,,,
+b10eae79-8358-4653-822b-33336644db36,,,Suspected TB symptom,រោគសញ្ញាសង្ស័យរបេង,Suspected TB symptom,រោគសញ្ញាសង្ស័យរបេង,,,Question,Coded,Yes;No,,,
```

Comment:
"Suspected TB symptom**s**" ? or should it be specifically singular?

Reply:
I may not have a right answer, however, I think it should be singular since there is one response(of either `yes` or `no`  basing on it's implementation. @rbuisson, do you think making it plural is a better way?

Reply:
I can't say what's the intent here. Sounds like this should even be: "Suspected TB".
Let's discuss with Khaled but if we don't get an answer right away, we merge as is.

Reply:
Done!